list

VolodymyrBabii75 opened the pull request.

DmitriyBabii committed:
add print

DmitriyBabii committed:
add print

DmitriyBabii committed:
Готово.

DmitriyBabii committed:
Створення списку користувачем
У програмі отримаєте рядок введений користувачем з клавіатури.

За допомогою циклу розбийте рядок на символи (окремі літери) і кожну окрему літеру додайте як новий елемент списку.

DmitriyBabii commented on lesson4/Створення списку користувачем.py:
Ти можеш зробити простіше, строка це особливий випадок масиву, це значить, що ти можеш привести тип даних строки в тип даних масиву

DmitriyBabii reviewed (commented):
Ти можеш зробити простіше, строка це особливий випадок масиву, це значить, що ти можеш привести тип даних строки в тип даних масиву

DmitriyBabii committed:
Створення списку користувачем
У програмі отримаєте рядок введений користувачем з клавіатури.

За допомогою циклу розбийте рядок на символи (окремі літери) і кожну окрему літеру додайте як новий елемент списку.

DmitriyBabii reviewed (approved):
Молодець
